examples/tutorials/sms/auto-subscribe-app/README.md

```diff
@@ -0,0 +1,59 @@
+# auto-subscribe application sample
+
+This directory contains sample related to Java SDK tutorials: [](https://developers.sinch.com/docs/sms/tutorials/sms/tutorials/java-sdk/auto-subscribe)
+
+## Requirements
+-
+- JDK 21 or later
+- [Maven](https://maven.apache.org/)
+- [ngrok](https://ngrok.com/docs)
+- [Sinch account](https://dashboard.sinch.com)
+
+## Usage
+
+### Configure application settings
+
+Application settings is using the SpringBoot configuration file: [`application.yaml`](src/main/resources/application.yaml) file and set:
+
+#### Sinch credentials
+Located in `credentials` section:
+- `key-id`: YOUR_access_key
+- `key-secret`: YOUR_access_secret
+- `project-id`: YOUR_project_id
```

Comment:
IMHO, `project-id` should be declared first as it is the top-level credentials we find in the dashboard.
It's already the case in the properties file and that's great! If you can update the readme in order to present the credentials always in the same order, it will be beneficial for the users

Reply:
Right.
Thank you it will provide a smooth transition when switching from dashboard to config, readme or sources